pkg/node/sendoperation/buyrolls/buy_rolls.go

```diff
@@ -47,3 +49,23 @@ func (b *BuyRolls) Message() []byte {
 
 	return append(msg, buf[:nbBytes]...)
 }
+
+func DecodeMessage(data []byte) (*OperationDetails, error) {
```

Comment:
This function looks exactly the same as the one in `sell_rolls`, shouldn't we move it in the `sendoperation` package to use the it at both places and avoid duplication ?

Reply:
if we move it, we should move it into a new file `rolls.go`